eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -92,6 +91,17 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.executeUpdate("CREATE INDEX metrics_name_idx ON audit.path_finding_metrics(experiment_name)")
       }
 
+      def migration910(statement: Statement): Unit = {
+        statement.executeUpdate("CREATE TABLE audit.transaction_published (tx_id TEXT NOT NULL PRIMARY KEY, channel_id TEXT NOT NULL, node_id TEXT NOT NULL, fee_sat BIGINT NOT NULL, tx_type TEXT NOT NULL, timestamp TIMESTAMP WITH TIME ZONE NOT NULL)")
+        statement.executeUpdate("CREATE TABLE audit.transaction_confirmed (tx_id TEXT NOT NULL PRIMARY KEY, channel_id TEXT NOT NULL, node_id TEXT NOT NULL, timestamp TIMESTAMP WITH TIME ZONE NOT NULL)")
```

Comment:
Done in https://github.com/ACINQ/eclair/pull/1976/commits/1ac5cc01aefb1d673b5ab9b7d1aedebd7107680d
I agree that `network_fees` was confusing, it's good that it's gone!
I think it makes sense to keep it very declarative like it is, it can then be used to extract the fees but also to just check on transactions published/confirmed that the channel should be concerned with.

> should we go with mining_fees

Do you mean for the field name inside the table (currently `fee`)?
Or the table name?
I think the table name is better as `transactions_published`, but I could rename the field to `miner_fee`.

Reply:
> Do you mean for the field name inside the table (currently fee)?

I mean the field. Did a quick poll here, people prefer `mining_fee`.

Reply:
Totally agree, it's more explicit. Done in https://github.com/ACINQ/eclair/pull/1976/commits/1bd8031a9c6bb2d47867c7ed7b175d97e6465763